config/ftbquests/quests/chapters/mekanism.snbt

```diff
@@ -471,8 +468,8 @@
 				{
 					id: "000000000000082C"
 					type: "item"
-					title: "Orange"
-					item: "simplefarming:orange"
+					title: "Melon"
+					item: "minecraft:melon_slice"
```

Comment:
No oranges in e8? Sad orange crush joke noises.

Just drop the melon entirely. The joke doesn't work :D

Reply:
Ah thats why it was here hah. I don't know those jokes unfortunetaly😅
How about we will add Enigmatica™️ Orange :D?

Reply:
(Nah I did delete melon completely, if some mod later on will add orange, I will re-add it here :D)